sunpy/net/tests/test_fido.py

```diff
@@ -90,13 +98,24 @@ def check_response(query, unifiedresp):
 
 @pytest.mark.remote_data
 def test_save_path():
+    qr = Fido.search(a.Instrument('EVE'), a.Time("2016/10/01", "2016/10/02"), a.Level(0))
+
+    # Test when path is str
     with tempfile.TemporaryDirectory() as target_dir:
-        qr = Fido.search(a.Instrument('EVE'), a.Time("2016/10/01", "2016/10/02"), a.Level(0))
         files = Fido.fetch(qr, path=os.path.join(target_dir, "{instrument}"+os.path.sep+"{level}"))
         for f in files:
             assert target_dir in f
             assert "eve{}0".format(os.path.sep) in f
 
+    # Test when path is pathlib.Path
```

Comment:
Shouldn't this go in its own test? It would be easier to manage and control when one or the other fails.